stable/prometheus-operator/values.yaml

```diff
@@ -357,6 +357,22 @@ grafana:
   #   configMap: certs-configmap
   #   readOnly: true
 
+  ## Configure additional grafana datasources
+  ## ref: http://docs.grafana.org/administration/provisioning/#datasources
```

Comment:
This content should also be added to the ci/ directory

Reply:
`additionalDataSources` added to ci dir